Tensile/KernelWriterAssembly.py

```diff
@@ -13444,7 +13444,7 @@ def globalWriteElements(self, kernel, vectorWidths, elements,
       if not self.do["PostLoop"]: return ""
     kStr = ""
     atomic = (kernel["GlobalSplitU"] > 1) and (kernel["_GlobalAccumulation"] != 'MultipleBuffer')
-    atomic = atomic or kernel["StreamK"] == 1
+    atomic = atomic or (kernel["StreamK"] > 0 and kernel["StreamKAtomic"] == 1)
```

Comment:
We can simplify this part. Just checking StreamKAtomic==1 is OK.

Reply:
Fixed